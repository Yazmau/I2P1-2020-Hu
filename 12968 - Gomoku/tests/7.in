100
..b............
........b...b..
....b.....b....
.w...........b.
...............
w..............
..b.w.....w....
...............
...............
....b..........
w......b..b....
.......w.......
.....w.........
..........w.w..
.......w.......
...............
......b........
...............
...............
...............
...............
www............
wbw............
wbw............
.b........b....
.b.............
.bw............
.w.............
.....w.........
...........bbbb
...............
...............
...............
...............
...............
...............
...............
...............
..............b
...........b...
...............
..w............
...bbb.........
w..bwwwwwb.....
...bbb......www
......b........
bbb............
bwb............
bbw............
...w...........
....w..........
.....w.........
......b........
.............b.
...............
...............
.....w.........
.............w.
...............
............www
.........www...
.........wbw...
.bb......bww.w.
........b......
.......b.......
......b........
.....w.........
...............
...............
...............
..........w....
...............
...............
...............
...........bbbb
...............
...............
....b...b....b.
b..............
..............b
w.............w
...........w...
.........b.....
...............
..w............
b...........bb.
.ww............
b............w.
w.......w......
............w.b
..b............
.......w.......
........b......
...............
...............
..........bbb..
..........bwb..
..........bwb..
...........w...
........w..w...
...........w...
...........b...
...............
...............
............www
............b..
...............
.....b.........
...............
...............
...............
...............
...............
...............
...............
.............w.
...............
.wbbb..........
..bwwwwwb......
..bbb.......www
...............
...b..w........
...............
...............
...............
...............
...............
...............
....bbb........
....bwb........
....bbw........
...b..ww.......
........w......
.........w.....
..........b.www
.........b.....
...............
.........w.....
...........b...
...............
...............
.........w.....
...............
............www
............wbw
............bww
...........b...
..........b....
.........b.....
........w..bbbb
......b..b.....
b..............
...b...........
...b...........
w..............
...............
...............
....b......b...
.....w....w....
..b.......b....
.....w..b...w..
...............
...w.w..w......
...............
...w.w........b
...............
...............
...........b...
...............
...............
.bbbb..........
.bwb...........
.bwb...........
..w............
..w............
..w............
..b..........w.
...............
...............
.w..........www
...............
...............
...............
...............
...............
...............
www............
wbbbbbw........
www............
b..............
..........b....
...............
......w........
w..............
...........bbbb
.....b.........
...............
..www..........
..wbw..........
..wwbb.........
.....b.........
......b........
.......b.......
........w......
...............
...............
..w....w.......
...............
...............
...........bbbb
...............
...............
..........b....
...............
.......b.......
......bbb......
......bwb......
......wbb......
.....w.........
....w..........
..ww......w....
..b............
...............
...............
............www
..b............
...............
....bb.........
.b.............
............w..
..b.....b..b...
...............
...............
....b..........
...w...........
.w.b...........
...w.......w.w.
w...w.....b....
.....w.........
..w............
...............
...............
...............
...b...........
.b.............
...............
...............
.......w.......
...www.........
...wbw.........
...wbw.........
....b..........
....b..........
....b......w...
....w......bbbb
.............b.
...............
...............
..w............
........www....
........wbbbbbw
........www....
...............
...............
...............
...............
...............
...b..........w
...............
...........bbbb
...............
...............
...............
...............
...............
.....bbb....b..
.....bwb.......
...w.bbw.......
....b...w.w....
.........w.....
..........w....
...........b...
...............
...............
............www
...............
...............
...............
b..............
.......bbbw....
.......bwb.....
.......wbb.....
......w.....b..
.....w.........
....w..........
...b...........
...w...........
...............
...............
............www
.........b.b...
...........b.w.
...............
....b..w.....w.
....b..........
..........b....
w..............
...............
b.........w....
.b.............
...w...........
....b..........
..b......w.....
w......w......w
..............b
...........b...
...............
...............
.......b.......
.www...........
.wbw..w........
.wbw...........
..b............
..b............
..b.....w......
..w............
...............
...............
...............
...........bbbb
...............
...............
..........b....
..b....www....w
.......wbbbbbw.
.......www.....
...............
...............
......w........
...............
...............
...............
...............
...............
...........bbbb
...............
........bbb....
........bwb....
........bbw....
...........w...
............w.b
.............w.
..............b
........b......
...............
.....w.........
...............
...............
...............
..w.........www
...............
..........bbb..
....b.....bwb..
..........wbb..
.........w.....
........w......
.......w.......
......b........
...............
...............
.............w.
...............
...............
......b........
..........w.www
..b............
...b..........b
..b....b.......
...............
........b......
............wb.
...............
.......b....wb.
...............
......w........
.....w.........
......w....w..b
............w..
w.......w......
..........w....
...............
...............
...............
...............
...............
........bbb....
........bwb....
........bwb....
.........w.....
..b......w.....
..w......w.....
.........b.....
..............b
...............
........w...www
...............
........b......
..........b....
...............
...............
..w............
...............
...............
..w.www........
....wbbbbbw....
....www........
...............
...............
...............
...........bbbb
............b..
...............
...............
....www........
b...wbw........
....wwb........
.......bw......
........b......
.........b.....
w.........w....
...............
...............
...............
...............
...........bbbb
......www......
......wbwb.....
......bww......
.....b.........
....b..........
...b...........
..w.....b......
...............
...............
........w......
...............
...............
...............
...............
.........w.bbbb
......b.....b..
...b...........
..........b.b..
...w...........
........w.b.w..
...............
..w.b..........
...............
..w............
..b...w......b.
...............
......bw.......
...w.w.........
...............
.w............b
...............
...............
...............
........b......
...............
..........b....
...............
.......www.....
.......wbw.....
...w...wbw.....
........b......
........b......
.......wb......
........w......
...........bbbb
...............
...............
...............
...............
...............
.....b.........
...............
........bbb....
........bwwwwwb
........bbb....
...............
.b.............
...w....w......
...............
............www
...............
...............
..www.b........
..wbw..........
..wwb..........
.....b.........
......b........
.......b.......
.......bw......
...............
...............
....w..........
....w..........
...............
...........bbbb
...............
...............
...............
.b.............
...............
...............
....wbbb..b....
.....bwb.......
.....wbb.......
....w..........
...w...........
.ww............
.b.............
...............
............www
.....b.........
...............
b...........b..
...............
.b....b........
.w....bb....b..
....bw........w
......w.....b..
...............
..............w
...............
.ww.....w......
....w..........
...............
.....w........b
...............
...............
........www....
........wbw....
........wbw...b
.........b.....
.........b.....
.........b.....
.........w.....
...............
..............b
...............
..............w
.........w.....
...........bbbb
...............
...............
...www.........
...wbbbbbw.....
..bwww.........
...............
...............
..b...........w
w..............
...............
...............
...............
...............
...............
...........bbbb
...............
...............
.........b.....
...............
...............
............w..
...............
....bbb.......b
....bwb........
....bbw........
.w.....w.......
........w......
.........w.....
..........b....
............www
...............
...............
..........b....
...............
........b......
...w...........
...............
......www......
......wbw......
......bww......
.....b.........
....b..........
...b...........
..w.....w......
...........bbbb
............b..
........b..b..b
...............
b..............
.......b.....w.
..b............
...............
..............w
...w..........w
......b....b...
b..............
.............w.
.......w....w..
..............w
..ww...........
...............
...............
.........b.....
...............
...............
...............
..........w..b.
......www......
......wbw......
......wbw......
.......b..w....
.......b.......
.......b.......
.......w.......
...........bbbb
...............
........b......
.......b.......
...............
...............
...............
bbb............
bwwwwwb........
bbb............
....w..........
...............
...............
...............
...............
......w.....www
...............
...............
...b...........
...b...........
...............
........bbb....
........bwb....
........bbw....
...........w...
...........ww..
w............w.
..............b
...............
...............
............www
...............
...............
...b...........
.w............b
...............
...............
...............
........bbb....
........bwb....
........wbb....
.......w.......
......w........
.....w.........
..w.b..........
............www
...b..........b
..............b
.....b.........
.w.............
....b..........
...............
b..............
.......w.......
..w.b..........
....b....w..w..
....b......w.b.
...............
.w.............
........w......
....w......w..b
...............
...............
...............
......bbb......
......bwb......
......bwb......
.......wb......
.......w.......
.......w....b..
.......b.......
...............
w..............
........w......
...............
............www
.bbb...b.......
.bwwwwwb.......
.bbb...........
....w..........
...............
...............
...............
...............
.......b.......
...............
...............
...............
...............
...............
........w...www
...............
..b............
...b...........
...............
...............
...............
........bbb....
........bwb....
........bbw....
...........w...
...........ww..
.............w.
..............b
...w...........
............www
...............
....b.....bbb..
..........bwb..
..........wbb..
.........w.....
........w......
.......w.......
......b........
.......b.......
...............
......w........
...............
...............
w..............
............www
...............
...............
...............
b..............
b..............
....b....b.b...
...........b...
......w.w......
.........b.....
..w............
....w....b.....
.w..w.........b
b.....w.w...w..
..............w
...............
...............
.....b........b
...............
...............
...............
...www.........
...wbw.........
...wbw.......w.
....b..........
....b..........
....b..........
....w..........
...............
...w...........
...........bbbb
...............
.......www.....
.......wbbbbbw.
...b...www.....
........w......
.....b.........
...............
...............
...............
...............
...............
..w............
...............
...............
...........bbbb
...............
bbbb...........
bwb............
bbw............
...w...........
....w..........
.....w.....w...
......b........
...............
...............
...........b...
...............
.............w.
...............
............www
...............
...............
...............
.......b.......
...............
...............
....www........
....wbw........
....bww........
...b...........
..b.........b..
.b........w....
w..............
...........w...
...........bbbb
....b..b...w...
...............
..b............
......b....b...
..b.....b......
...............
......w......b.
.wb............
.............b.
...ww..........
.....ww........
..w............
............w..
...............
.........w....b
...............
...............
...............
...........b...
...............
.........bbb...
.........bwb.b.
.........bwb...
..........w....
..........w....
..w.......w....
..........b....
...............
....w..........
............www
...............
......b........
...............
...............
...............
...www.........
...wbbbbbw.w...
...www.........
...............
...............
........b......
...............
...............
...............
...w.......bbbb
...............
...............
...............
...............
...............
bbb............
bwb............
bbw............
.b.w...........
....w..........
.....w....b....
......b........
...w....w......
...............
............www
..........b....
...............
...............
...............
...........bbb.
....b......bwb.
..w........wbb.
..........w...w
.........w.....
........w......
.......b.......
...............
...............
...............
............www
.b...b..b..b..b
..............b
...............
..b.w...w......
..............w
...........b...
...............
............w..
...w...........
...............
.........w.....
......w.b......
....w..........
b..w.........w.
..............b
.....www.......
.....wbw.......
.....wbw.......
...b.wb........
......b........
......b........
......w........
...b...........
.............w.
...............
...............
...............
...............
...............
...........bbbb
...............
bbb............
bwwwwwb........
bbb............
.........b.....
b..............
...............
...............
...............
...............
...............
w..............
.............w.
...............
............www
...............
...b...........
....b......w...
...............
...............
...............
.......w.......
.....www.......
.....wbw.......
.....wwb.......
........b......
.........b.....
..........b....
...........w...
...........bbbb
...............
...............
...............
...............
.......b.......
.........www...
.........wbw...
.........bww...
.b......b......
.......b.......
......b.w......
.....w.........
...............
.....w.........
...........bbbb
.............b.
b..............
..bb...........
w..........bw..
...........bb..
.w.......w.....
.......bb......
...............
........b......
...............
......w........
...............
..............w
..........ww.w.
...........w...
...............
.bbb...........
.bwb...........
.bwb...........
.bw............
..w............
..w............
..b............
...............
...............
.....w.........
...........b...
...............
...........w...
............www
..............b
...............
...............
...............
...............
...............
......w........
...............
.......www.....
.......wbbbbbw.
.......www.....
..b..........w.
...............
...............
...........bbbb
...............
...............
...............
...............
.....b.........
...............
..............b
..............w
....bbb........
....bwb.w......
....bbw........
.......w.......
........w......
.........w.....
..........b.www
...............
...............
....b..........
........b......
...............
...............
...........w...
...............
...........bbb.
..w........bwb.
...........wbb.
..........w....
.........w.....
........w......
.......b....www
.b............b
...............
b.....b.....b..
..w...b..w.....
....b..b.......
...w...........
...............
...............
...............
.....b.........
...w...........
........b.....w
...........w...
.............w.
.....ww..w....b
...............
...............
....www........
....wbw........
....wbw........
.....b.........
.....b......b..
.....b.........
.....w.........
..b............
...............
...........w...
............w..
...............
...........bbbb
...............
...........b...
...............
...............
......b........
..w............
...............
.............w.
...............
...............
...bbb.........
...bwwwwwb.....
...bbb.........
...............
............www
......b....b.w.
....w..........
...............
...............
...............
...............
...............
...............
......www......
......wbw......
......wwb......
.........b.....
..........b....
...........b...
..........bbwbb
...............
...............
...............
...............
..........bbb..
..........bwb..
..........wbb..
.........w.....
.....b..w......
.......w.......
......b........
..........b....
...............
.w........w....
............www
..b...........b
..b............
..............b
..............w
......bw.......
...............
...b.......b...
.b.............
........w......
...w..........w
b...b..........
...............
..w............
.....w...w....w
.......w......b
www............
wbw............
wbw.b..........
.b........b....
.b.............
.b.............
.w.............
......w....w...
...............
...............
...............
...............
...............
...............
...........bbbb
...............
...b...........
.............w.
...............
...............
...............
...............
......bbb......
......bwwwwwb..
b..w..bbb......
...............
...............
...............
...............
............www
...............
...............
...............
...b...........
..............b
...............
...............
.bbb...........
.bwb...........
.bbw...........
....w...w......
.....w.........
......w....w...
.......b.......
............www
...............
...............
...............
...............
...............
......www......
......wbw.....b
......bww......
.....b.........
....b..........
.b.bw.w........
..w............
...............
...............
...........bbbb
.........b.....
..b............
.........b..b..
..............w
........b......
...............
.b......b......
.....w.........
..............b
.w...b.....w..b
.w.............
...............
.......w..ww..w
w..............
..............b
...............
..........b....
...............
b..............
.....bbb.......
.....bwb.......
.....bwb.....w.
......w........
......w........
......w........
......b...w....
...............
...............
...............
............www
...............
.b.............
...............
...............
...............
.........w.....
.b.............
...............
...............
.....www.......
.....wbbbbbw...
.....www.......
...............
..w............
...........bbbb
...............
...............
........www....
........wbw....
........wwb....
...........b...
............b..
.............b.
..............w
...............
...............
......b...b....
...............
.......w..w....
...........bbbb
.b.............
...............
......www......
......wbw......
......bww......
...w.b.........
....b..........
...b...........
b.w............
...............
...............
........w......
...............
...............
...........bbbb
...............
........b......
.............b.
...............
..b............
...............
b..............
........w....b.
bw.......b.....
........bb.....
.........w..b.w
...............
.......w.......
...............
.w.w...w.w..w.b
...............
...............
..........b....
...............
..........b....
...............
.........www...
.........wbw...
.........wbw...
..........b....
..........b....
........w.b....
..........w..w.
...............
...........bbbb
.....b.........
...............
...www.........
...wbbbbbw.....
.b.www.........
...............
...........w...
...............
.......w.......
...............
...............
...............
...............
...............
...........bbbb
...............
...........b...
...............
..............b
.......w..w....
...............
...............
.www...........
.wbw...........
.wwb...........
....b..........
.....b.........
......b........
.......w.......
...........bbbb
............b..
...............
.........w.....
...............
...............
...............
...............
............bbb
............bwb
............wbb
...........w...
..........w....
.......b.w.....
......w.b......
............www
........b......
...........b...
...b...........
...............
..........w....
....b..........
......b....b..b
....w.........b
....b..b.......
...............
.w.............
w.w............
...............
w....w.....ww.w
..............b
...............
....b..........
...............
...........b...
.......w.......
...............
...............
...............
..........www..
..........wbw..
..........wbw..
...........b...
..........wb...
...........b...
..........bwbbb
...............
..b............
...........b...
.www...........
.wbbbbbw.......
.www...........
.......w.......
...............
...............
...............
...............
...............
...............
...........w...
...........bbbb
...............
...............
.............b.
...............
www...b........
wbw............
wwb.......w....
...b...........
....b...w......
.....b.........
......w........
...............
...............
...............
...........bbbb
............bbb
............bwb
............wbb
....b......w...
..........w....
.........w.....
........b......
...............
...............
.......w.......
.....b.........
...............
..........w....
...............
............www
...............
.....b.........
b........b.b...
.............bw
..b............
...............
..w...........b
...............
..w......bw....
........b..w...
.........w..w.w
.....w.........
...............
...............
b........w.....
...............
...............
..b............
...............
...............
...............
b.....w........
...............
...........bbb.
...........bwb.
...........bwb.
............w..
....w.......w..
............w..
...........wbww
...............
...............
.........b.....
...............
...............
...............
.b.............
...............
...............
...............
...............
...............
......bbb...w..
......bwwwwwbw.
......bbb...www
...............
...............
...............
...............
....www........
....wbw........
....wwb........
.......b.......
........b......
.........b.....
..........w....
.bb............
..........w....
............w..
...........bbbb
...............
.........bbb...
.........bwb...
.........wbb...
........w......
.......wb......
......w........
.....b.........
...............
...............
..b.......w.w..
...............
...............
...............
............www
