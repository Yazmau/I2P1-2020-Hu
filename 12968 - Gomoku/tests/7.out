none
black
white
white
black
none
white
white
white
black
none
white
black
black
white
none
black
black
white
white
none
black
black
white
white
none
white
black
black
black
none
black
white
black
white
none
black
black
white
black
none
black
white
white
white
none
white
white
white
white
none
black
black
white
black
none
white
black
white
white
none
black
white
black
black
none
white
black
white
white
none
black
white
black
white
none
black
white
white
black
none
white
black
black
black
none
black
black
black
white
none
black
black
black
white
none
white
white
black
white
